### PR TITLE
removed duplicate import

### PR DESCRIPTION
**Pull request by sureshdsk**
**Proposed changes**:
- Remove duplicate imports in shared/importers/importer.py
- Resolves issue #7015 

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @melindaloubser1 will take a look at it as soon as possible ✨

File: changelog/7015.misc.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove duplicate imports in shared/importers/importer.py
```
Comment (melindaloubser1) [inline changelog/7015.misc.md:1]:
```suggestion
Remove duplicate import of ``ActionExecuted`` from  ``shared.importers.importer``
```

**Review by melindaloubser1 — APPROVED** [review]
Thanks for the PR! Just updating the changelog to be more specifc.

**Comment by sureshdsk** [comment]
@melindaloubser1 updated the branch.

**Pull request merged**